CMakeLists.txt

```diff
@@ -348,22 +348,47 @@ if (USE_NCURSES)
 	find_package(Curses)
 	CHECK_INCLUDE_FILE(term.h HAVE_TERM_H)
 	CHECK_INCLUDE_FILE(ncurses/term.h HAVE_NCURSES_TERM_H)
-	if (CURSES_FOUND AND (HAVE_TERM_H OR HAVE_NCURSES_TERM_H) AND (CURSES_HAVE_CURSES_H OR CURSES_HAVE_NCURSES_CURSES_H))
-		# HACK: We should use ${CURSES_LIBRARIES} here, but that breaks static
-		# builds as it contains the full path to ncurses.so
-		target_link_libraries(cfunge ncurses)
-		add_definitions(-DHAVE_NCURSES)
-		if (HAVE_NCURSES_TERM_H)
-			add_definitions(-DTERM_H_IN_NCURSES)
-		endif ()
-		if (CURSES_HAVE_NCURSES_CURSES_H)
-			add_definitions(-DCURSES_H_IN_NCURSES)
-		endif ()
-		if (CURSES_HAVE_NCURSES_NCURSES_H)
-			add_definitions(-DNCURSES_H_IN_NCURSES)
-		endif ()
-	else ()
-		message(STATUS "ncurses and/or term.h not found: disabling the TERM fingrprint.")
+
```

Comment:
You have changed from using tabs for indentation to spaces. This breaks the diff making it hard to tell what changed. And it also breaks the coding style used in cfunge (tabs for indentation, spaces for adjustment in case of aligning a line that has been broken across multiple).

Reply:
Sorry about that, I'll fix it.

The diff will remain hard to read though, as I've indented a block one indentation step further in.